### PR TITLE
1625/fix/change-county-return-to-default-page

### PR DESCRIPTION
**Pull request by Noam6421**
# NOTE: SPRINT 16
Fix to bug: [1625](https://dev.azure.com/spectrumFactory/CoronaI/_workitems/edit/1625/)

### CODE REVIEW

**Review by barnahum8 — APPROVED** [review]

**Review by chenFriedman — APPROVED** [review]

File: client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts [hunk 1]
```diff
@@ -571,7 +571,15 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
             fetchTableData();
         }
         setIsBadgeInVisible(!Boolean(Object.values(filterRules).find(item => item !== null)))
-    }, [isLoggedIn, filterRules, orderBy, currentPage, displayedCounty, userType]);
```
Comment (y-golde) [inline client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts:574]: are you sure these 2 need to be removed form here? why not just add a new useEffect with setCurrentPage(defaultPage); ?

File: client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts [hunk 2]
```diff
@@ -571,7 +571,15 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
...
+    }, [isLoggedIn, filterRules, orderBy, currentPage]);
+
+    useEffect(() => {
+        setCurrentPage(defaultPage);
+        if (isLoggedIn) {
```
Comment (y-golde) [inline client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts:578]: Just looked inside fetchTableData - and isLoggedIn is already in it, is it neccecary to check here also?

**Review by y-golde — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts (hunk 1 above)
Comment (Noam6421) [reply client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts:574]: changed it like you said

File: client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts (hunk 2 above)
Comment (Noam6421) [reply client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts]: removed the if

**Review by y-golde — APPROVED** [review]

**Review by chenFriedman — APPROVED** [review]

**Pull request merged**